crates/parser/src/jup_programs/raydium/ix_parser/parser.rs

```diff
@@ -0,0 +1,136 @@
+use std::borrow::Cow;
+
+use borsh::BorshDeserialize;
+use yellowstone_vixen_core::{
+    instruction::InstructionUpdate, ParseError, ParseResult, Parser, Prefilter, ProgramParser,
+};
+
+use super::{
+    RaydiumProgramIx, SwapAccounts, SwapIxData, SwapV2Accounts, SWAP_IX_DISC, SWAP_V2_IX_DISC,
+};
+use crate::{
+    helpers::{check_min_accounts_req, IX_DISCRIMINATOR_SIZE},
+    jup_programs::raydium::RADIUM_V3_PROGRAM_ID,
+};
+
+#[derive(Debug, Clone, Copy)]
+pub struct RaydiumProgramIxParser;
+
+impl Parser for RaydiumProgramIxParser {
+    type Input = InstructionUpdate;
+    type Output = RaydiumProgramIx;
+
+    fn id(&self) -> Cow<str> {
+        "yellowstone_vixen_parser::token_program::RaydiumProgramIxParser".into()
```

Comment:
```suggestion
impl Parser for radium::InstructionParser {
    type Input = InstructionUpdate;
    type Output = RaydiumProgramIx;

    fn id(&self) -> Cow<str> {
        "yellowstone_vixen_parser::token_program::RaydiumProgramIxParser".into()
```

I'm been thinking about mod structure for the parser and think we should expose all pub on the root module in this case `radium` and then name have consistent parser names so `InstructionParser` and `AccountParser`

Reply:
Is this ID correct? token_program?

`yellowstone_vixen_parser::jup::radium::InstructionParser`

Reply:
will be doing this in a new pr